src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -104,13 +111,12 @@ public String getColumnValue(final VariantContext variant) {
                 } else {
                     outValue = outNotAlleleSpecificAndOld;
                 }
-                // Spread MQ accross multiple alleles.
-                //TODO: check how hail does this and change this to do the same thing
+
                 double mq = Double.parseDouble(outValue);
                 if (variant.getAlleles().size() == 3) {
-                    outNotAlleleSpecific = (int) mq / 2 + VCFConstants.PHASED + (int) mq / 2;
+                    outNotAlleleSpecific = (int) mq + VCFConstants.PHASED + (int) mq;
```

Comment:
Please use AnnotationUtils.ALLELE_SPECIFIC_RAW_DELIM instead of VCFConstants.PHASED

Reply:
good catch